portfolio/src/main/webapp/script.js

```diff
@@ -16,7 +16,20 @@
  * Adds a random greeting to the page.
  */
 
-$(document).ready(initClasses);
+$(document).ready(init);
+
+function init(){
+  initCardImgs();
+  initClasses();
+}
+
+function initCardImgs(){
+  $(".card-img").each( function(){
+    src = 'url("images/' + $(this).attr('id') +'.jpg")';
+    $(this).css("background-image", src);
```

Comment:
You could extract `$(this)` into a separate variable so you don’t have to access it multiple times.

Reply:
It's a function call, so it always have some performance hit. Probably overhead is marginal in this situation, but good practice is to refactor repeated calls to the same function (if the expected return value should be the same and function does not have some side effects).